torch/distributions/inverse_gamma.py

```diff
@@ -38,7 +38,7 @@ class InverseGamma(TransformedDistribution):
         "concentration": constraints.positive,
         "rate": constraints.positive,
     }
-    support = constraints.positive
+    support: ClassVar[constraints.Positive] = constraints.positive  # type: ignore[assignment]
```

Comment:
Apparently, this fails test typing because constraints.positive is typed as Any

Reply:
No, this is referring to `/test/typing/pass/distributions.py`, I must have messed up somehow because it was actually me who added these checks there.

Reply:
Should be fixed now.